### PR TITLE
Feat: Add undeploy service command

### PR DESCRIPTION
**Pull request by umanggoyald11**
# Odin version(x.y.z)

### CODE REVIEW

File: proto/gen/go/dream11/od/auth/v1/auth.pb.go [hunk 1]
```diff
@@ -1,7 +1,7 @@
 // Code generated by protoc-gen-go. DO NOT EDIT.
 // versions:
 // 	protoc-gen-go v1.33.0
-// 	protoc        v5.26.1
```
Comment (boppanasusanth) [inline proto/gen/go/dream11/od/auth/v1/auth.pb.go:4]: install the latest protoc version. use `brew install protoc`

File: proto/dream11/od/service/v1/service.proto [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+}
+
+message ComponentStatus {
+  string component_name = 1;
+  string component_status = 2;
+  string component_action = 3;
+  string error = 4;
+}
+
+message DeleteEnvironmentRequest {
```
Comment (boppanasusanth) [inline proto/dream11/od/service/v1/service.proto:34]: These are not needed here

File: internal/service/service.go [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+	}
+
+	client := service.NewServiceServiceClient(conn)
+	stream, err := client.UndeployService(*requestCtx, request)
+
+	if err != nil {
+		return err
+	}
+
+	log.Info("Deleting environment...")
```
Comment (boppanasusanth) [inline internal/service/service.go:31]: change the log to `Undeploying service...`

File: proto/dream11/od/service/v1/service.proto [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+  string env_name = 1;
+}
+
+message DeleteEnvironmentResponse {
```
Comment (boppanasusanth) [inline proto/dream11/od/service/v1/service.proto:38]: same as above. not needed

File: proto/dream11/od/dto/v1/service.proto [hunk 5]
```diff
@@ -0,0 +1,23 @@
+syntax = "proto3";
+package dream11.od.dto.v1;
+
+import "dream11/od/dto/v1/service_task.proto";
+import "google/protobuf/struct.proto";
+import "google/protobuf/timestamp.proto";
+
+option go_package = "github.com/dream11/odin/proto/gen/go/dream11/od/dto/v1";
+
+message Service {
```
Comment (boppanasusanth) [inline proto/dream11/od/dto/v1/service.proto:10]: This is not the proto for service as far as I know. where did you get this?

**Review by boppanasusanth — DISMISSED** [review]

**Review by dhavalmehta07 — DISMISSED** [review]

**Review by boppanasusanth — APPROVED** [review]

**Pull request merged**